Client/Frontend/Browser/Onboarding/OnboardingRewardsView.swift

```diff
@@ -0,0 +1,132 @@
+            $0.accessibilityIdentifier = "OnboardingRewardsViewController.JoinButton"
+        }
+        
+        let skipButton = CommonViews.secondaryButton().then {
+            $0.accessibilityIdentifier = "OnboardingRewardsViewController.SkipButton"
+        }
+        
+        private let mainStackView = UIStackView().then {
+            $0.axis = .vertical
+            $0.spacing = UX.negativeSpacing
+        }
+        
+        let imageView = AnimationView(name: "onboarding-rewards").then {
+            $0.contentMode = .scaleAspectFit
+            $0.setContentCompressionResistancePriority(.defaultLow, for: .vertical)
+            $0.backgroundBehavior = .pauseAndRestore
+            $0.loopMode = .loop
+            $0.play()
+        }
+        
+        private let descriptionView = UIView().then {
+            $0.layer.cornerRadius = 12
+            $0.layer.maskedCorners = [.layerMinXMinYCorner, .layerMaxXMinYCorner]
+        }
+        
+        private let descriptionStackView = UIStackView().then {
+            $0.axis = .vertical
+            $0.spacing = 32
+        }
+        
+        private let textStackView = UIStackView().then { stackView in
+            stackView.axis = .vertical
+            stackView.spacing = 8
+            
+            let titleLabel = CommonViews.primaryText(Strings.OBRewardsTitle)
+            
+            let descriptionLabel = CommonViews.secondaryText("").then {
+                $0.attributedText = Strings.OBRewardsDetail.boldWords(with: $0.font, amount: 2)
+            }
+            
+            [titleLabel, descriptionLabel].forEach {
+                stackView.addArrangedSubview($0)
+            }
+        }
+        
+        private let buttonsStackView = UIStackView().then {
+            $0.distribution = .equalCentering
+        }
+        
+        init(theme: Theme, themeColour: UIColor) {
+            super.init(frame: .zero)
+            
+            descriptionView.backgroundColor = themeColour
+            
+            mainStackView.tag = OnboardingViewAnimationID.details.rawValue
+            descriptionStackView.tag = OnboardingViewAnimationID.detailsContent.rawValue
+            imageView.tag = OnboardingViewAnimationID.background.rawValue
+            
+            addSubview(imageView)
+            addSubview(mainStackView)
+            mainStackView.snp.makeConstraints {
+                $0.leading.equalTo(self.safeArea.leading)
+                $0.trailing.equalTo(self.safeArea.trailing)
+                $0.bottom.equalTo(self.safeArea.bottom)
+            }
+            
+            descriptionView.addSubview(descriptionStackView)
+            descriptionStackView.snp.makeConstraints {
+                $0.edges.equalToSuperview().inset(UX.descriptionContentInset)
+            }
+            
+            [descriptionView].forEach(mainStackView.addArrangedSubview(_:))
+
+            [skipButton, joinButton, UIView.spacer(.horizontal, amount: 0)]
```

Comment:
No spacers, same as above

Reply:
All of these spacers are currently in dev. I just copied all of them to keep the code consistent: https://github.com/brave/brave-ios/search?utf8=%E2%9C%93&q=UIView.spacer%28.vertical&type=

Even on the old on-boarding code it is there.